composer.json

```diff
@@ -4,10 +4,10 @@
     "description": "Google Analytics integration for Neos CMS",
     "license": "GPL-3.0-or-later",
     "require": {
-        "neos/neos": "^7.0 || dev-master",
-        "neos/fusion-afx": "^7.0",
+        "neos/neos": "^7.0 || ^8.0 || dev-master",
+        "neos/fusion-afx": "^7.0 || ^8.0",
```

Comment:
```
        "neos/fusion-afx": "^7.0 || ^8.0",
```

This could be removed, as it is included in `neos/neos`

Reply:
True, but "transitive vs explicit dependencies"… I am fine with both ways, a long as this can be used with Neos 8. 😎